### PR TITLE
Add Breadcrumb Tests. Add Decline ToS Url. Update unit tests.

### PR DESCRIPTION
**Pull request by dimak1**
*Issue:*

*Description of changes:*
- Add Breadcrumb tests
- Add Decline Terms of Service redirect URL 
- Update test dependencies/packages

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-144-b082r8ll.web.app

**Review by shaangill025 — APPROVED** [review]

File: strr-web/package.json [hunk 1]
```diff
@@ -44,7 +45,7 @@
     "sass-loader": "^13.3.3",
     "typescript": "5.3.3",
     "vite": "5.4.6",
-    "vitest": "2.1.1",
+    "vitest": "1.3.1",
```
Comment (rstens) [inline strr-web/package.json:48]: That is a significant drop in **vitest** version, why?

**Review by rstens — COMMENTED** [review]
So many version updates on so many components - would almost need a full regression test. the pnpm lockfile has many changes. Were they all necessary or were they simply due?

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-144-b082r8ll.web.app

File: strr-web/package.json (hunk 1 above)
Comment (dimak1) [reply strr-web/package.json:48]: I reverted back to what it was prior my recent PR.

**Comment by rstens** [comment]
image

The delimiters ">" are missing, according to Andy they were there before.

**Comment by dimak1** [comment]
@rstens Re Breadcrumb delimiters, try refreshing? Is it persistent on actual Dev/Test env?

**Comment by rstens** [comment]
> @rstens Re Breadcrumb delimiters, try refreshing? Is it persistent on actual Dev/Test env?

Yes, it persistent, tried incognito browser - same result.

**Comment by dimak1** [comment]
@rstens icons not loading due to error below, will look into it

Screenshot 2024-09-18 at 16 57 49

**Comment by dimak1** [comment]
@rstens regarding the icons issue, it seems only in the preview mode they are not showing up (see below), in Dev it looks ok.
Screenshot 2024-09-19 at 04 22 59

File: strr-web/package.json [hunk 2]
```diff
@@ -43,8 +44,8 @@
     "sass": "^1.71.1",
     "sass-loader": "^13.3.3",
     "typescript": "5.3.3",
-    "vite": "5.2.0-beta.1",
-    "vitest": "1.3.1",
+    "vite": "5.4.6",
+    "vitest": "1.6.0",
```
Comment (dimak1) [inline strr-web/package.json:48]: @rstens I updated the version up until the breaking changes were introduced. Any further updates would require a migration.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-144-b082r8ll.web.app

**Comment by rstens** [comment]
@dimak1 Could you take a quick look at the new SonarCloud issues (6), they are all classified as medium or low that's why it is not failing. I am just thinking if this needs to be cleaned up or labeled "Accepted"

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-144-b082r8ll.web.app

**Pull request merged**